packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessibilityGuide.2.doc.mdx

```diff
@@ -79,14 +79,14 @@ You should pick an `a11yLabel` that conveys the same information that a user vie
     <A11yMeaningfulReadOnly />
 </div>
 <div aria-label="Example code">
-
+```js
```

Comment:
I guess here something went wrong with formatting as it looks like this (see highlited):
<img width="552" alt="Screenshot 2023-09-20 at 4 57 29 PM" src="https://github.com/cerner/terra-core/assets/119358186/194737db-0c59-4f3e-a05c-51d343a29083">

Reply:
This is an issue that already exists in main:

![CleanShot 2023-09-20 at 17 50 01](https://github.com/cerner/terra-core/assets/38024451/2db9420b-c292-4868-b2c7-ad343522dece)

The important tag is being misused here and the document needs to be rewritten, which while likely happen as part of the documentation uplift project.

CC: @mjpalazzo , something to take note of that will need fixing.

Reply:
Correction, this was resolved in: 4533e1b000d7800c324494d791e8eedce0463125